### ISSUE TITLE
HEALTHCHECK failure - curl does not exist in container

### ISSUE THREAD

**Issue by slaffer-au** [fescobar/allure-docker-service-ui#12]
The container does not have curl installed, so the Docker HEALTHCHECK fails and marks the containter as unhealthy.

```
$ docker ps -a
CONTAINER ID        IMAGE                                   COMMAND                  CREATED             STATUS                      PORTS                              NAMES
afe740aec774        frankescobar/allure-docker-service      "/bin/sh -c '$ROOT/r…"   12 minutes ago      Up 12 minutes (healthy)     4040/tcp, 0.0.0.0:5050->5050/tcp   allure-docker-service_allure_1
137514e8a711        frankescobar/allure-docker-service-ui   "docker-entrypoint.s…"   12 minutes ago      Up 12 minutes (unhealthy)   0.0.0.0:5252->5252/tcp             allure-docker-service_allure-ui_1
```

```
$ docker inspect allure-docker-service_allure-ui_1 | grep -A 20 Health
            "Health": {
                "Status": "unhealthy",
                "FailingStreak": 77,
                "Log": [
                    {
                        "Start": "2021-01-04T18:53:08.884038808Z",
                        "End": "2021-01-04T18:53:09.03943869Z",
                        "ExitCode": 1,
                        "Output": "/bin/sh: 1: curl: not found\n"
                    },
                    {
                        "Start": "2021-01-04T18:53:19.043791799Z",
                        "End": "2021-01-04T18:53:19.20149147Z",
                        "ExitCode": 1,
                        "Output": "/bin/sh: 1: curl: not found\n"
                    },
                    {
                        "Start": "2021-01-04T18:53:29.205670364Z",
                        "End": "2021-01-04T18:53:29.377821007Z",
                        "ExitCode": 1,
                        "Output": "/bin/sh: 1: curl: not found\n"
--
            "Healthcheck": {
                "Test": [
                    "CMD-SHELL",
                    "curl -f http://localhost:$PORT || exit 1"
                ],
                "Interval": 10000000000,
                "Timeout": 60000000000,
                "Retries": 3
```

**Comment by fescobar** [comment]
Thanks @slaffer-au . Added to the backlog.

**Comment by fescobar** [comment]
@slaffer-au The improvement was released in the `7.0.2` version.
https://github.com/fescobar/allure-docker-service-ui/releases/tag/v7.0.2
I appreciate too much your help. Thank you.
